uwrt_mars_rover_blackfly_calibration/uwrt_mars_rover_blackfly_calibration/broadcaster.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+import rclpy
+from rclpy.node import Node
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image
+import EasyPySpin
```

Comment:
i also saw stevens camera branch which had some code for the blackfly. is that branch completed and tested. if not, its prob easier to use the python lib for the camera driver than to debug his code?

Reply:
which branch? i know he was doing something about using webrtc to get around some of ros1's shitty compression but i dont think it was every fully finished

Reply:
this one https://github.com/uwrobotics/uwrt_mars_rover/tree/cameras i see a pkg called spinnaker cmaera driver